scripts/create-darwin-volume.sh

```diff
@@ -463,7 +463,7 @@ EOF
 
     EDITOR="$SCRATCH/ex_cleanroom_wrapper" _sudo "to add nix to fstab" "$@" <<EOF
 :a
-UUID=$uuid $escaped_mountpoint apfs rw,noauto,nobrowse,suid,owners
+UUID=$uuid $escaped_mountpoint apfs rw,noauto,nobrowse,nosuid,noatime,owners
```

Comment:
cc @risicle You might need to update documentation for macOS users in this case if they want to use your: https://github.com/risicle/nix-heuristic-gc/